package.json

```diff
@@ -26,13 +29,16 @@
     "coverage:ci": "node packages/build/bin/run-nyc report --reporter=text-lcov | coveralls",
     "precoverage": "npm test",
     "coverage": "open coverage/index.html",
-    "lint": "npm run prettier:check && npm run tslint",
-    "lint:fix": "npm run prettier:fix && npm run tslint:fix",
+    "lint": "npm run prettier:check && npm run tslint && npm run lint-md",
+    "lint:fix": "npm run prettier:fix && npm run tslint:fix && npm run lint-md:fix",
+    "lint-md": "remark -f -q .",
```

Comment:
curious to know what constitutes as warnings and if we really want to fail on them

Reply:
You can see the style preset I am using here https://github.com/remarkjs/remark-lint/tree/master/packages/remark-preset-lint-markdown-style-guide#rules

And the rules I've disabled / changed from the preset can be found in `.remarkrc`

We can always tweak the rules as needed. I'm open to suggestions. 

An example of a rule that's enforced and I liked was the code fences (three backticks) MUST have a type (`ts`, `js`, `json`, `text`, `sh`, etc).

Reply:
Thank you for clarifying! If that is the type of rules that are counted as warnings, I am fine failing on them.